packages/terra-table/src/subcomponents/_SectionHeader.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import {
+  wrappedOnClickForItem,
+  wrappedOnKeyDownForItem,
+  wrappedEventCallback,
+} from './utils';
+import styles from './SectionHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The colSpan to be used as a descriptor for assistive technology.
+   */
+  colSpan: PropTypes.number,
+  /**
+   * The id to be place on the column header.
+   */
+  id: PropTypes.string,
+  /**
+   * @private Whether or not the section is collapsed.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * @private Whether or not the section can be collapsed.
+   */
+  isCollapsible: PropTypes.bool,
+  /**
+   * Optionally sets the heading level.
+   */
+  level: PropTypes.oneOf([2, 3, 4, 5, 6]),
+  /**
+   * @private The associated metaData to be provided in the onSelect callback.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  metaData: PropTypes.object,
+  /**
+   * The numberOfColumns to be used as a descriptor for assistive technology.
+   */
+  numberOfColumns: PropTypes.number.isRequired,
+  /**
+   * @private Function callback for when the appropriate click or key action is performed.
+   * Callback contains the javascript event and prop metadata, e.g. onSelect(event, metaData)
+   */
+  onSelect: PropTypes.func,
+  /**
+   * Function callback returning the html node of the header.
+   */
+  refCallback: PropTypes.func,
+  /**
+   * Title text to be placed within the section header.
+   */
+  title: PropTypes.string.isRequired,
+  /**
+   * @private Callback function not intended for use with this API, but if set pass it through to the element regardless.
+   */
+  onBlur: PropTypes.func,
+  /**
+   * @private Callback function not intended for use with this API, but if set pass it through to the element regardless.
+   */
+  onClick: PropTypes.func,
+  /**
+   * @private Callback function not intended for use with this API, but if set pass it through to the element regardless.
+   */
+  onKeyDown: PropTypes.func,
+  /**
+   * @private Callback function not intended for use with this API, but if set pass it through to the element regardless.
+   */
+  onMouseDown: PropTypes.func,
+};
+
+const defaultProps = {
+  isCollapsed: false,
+  isCollapsible: false,
+  level: 2,
+};
+
+const SectionHeader = ({
+  isCollapsed,
+  isCollapsible,
+  level,
```

Comment:
I don't see where level is used in this file.

Reply:
Removed.